docs/structure.md

```diff
@@ -0,0 +1,156 @@
+(structure-page)=
+
+# Structure
```

Comment:
What does "structure" mean?

Reply:
"Structure" of the project directory. Information about the project-level `bin` and `lib` directories currently sits on the [Sharing pipelines](https://www.nextflow.io/docs/latest/sharing.html) page, and it's hard to find.

Maybe "Project directory" would be better? Or something else?

Reply:
I think "Project Structure" would be best